lib/bitmap/test_bitmap.nit

```diff
@@ -0,0 +1,44 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Copyright 2015 Budi Kurniawan <budi2020@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This file tests the Bitmap class in the bitmap module
+# It has to be called by passing two arguments:
+#
+#	- An input bitmap file
+#	- An output bitmap file
```

Comment:
I've implemented all the required changes.

On Mon, May 11, 2015 at 10:12 AM, Jean Privat <notifications@github.com>
wrote:

> In lib/bitmap/test_bitmap.nit
> <https://github.com/privat/nit/pull/1282#discussion_r30041302>:
>
> > +# you may not use this file except in compliance with the License.
> > +# You may obtain a copy of the License at
> > +#
> > +#     http://www.apache.org/licenses/LICENSE-2.0
> > +#
> > +# Unless required by applicable law or agreed to in writing, software
> > +# distributed under the License is distributed on an "AS IS" BASIS,
> > +# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +# See the License for the specific language governing permissions and
> > +# limitations under the License.
> > +#
> > +# This file tests the Bitmap class in the bitmap module
> > +# It has to be called by passing two arguments:
> > +#
> > +#	- An input bitmap file
> > +#	- An output bitmap file
>
> I am not sure if <tab>- is seend as a list item in markdown. try to use 3
> spaces instead.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/privat/nit/pull/1282/files#r30041302>.
>